lib/rubocop/config_loader.rb

```diff
@@ -134,6 +134,12 @@ def default_configuration
                                    end
       end
 
+      # Returns the path rubocop inferred as the root of the project. No file
+      # searches will go past this directory.
+      def project_root
+        @project_root ||= find_project_root
+      end
+
```

Comment:
I intentionally left this method in the public section and documented it since I believe it could be useful for extensions. At least it's useful for [a cop I'm helping with in the `rubocop-packaging` extension](https://github.com/utkarsh2102/rubocop-packaging/pull/4).